### DISCUSSION TITLE
Sin trào các ty của iem

### DISCUSSION THREAD

**Discussion by GiaHan06** [GiaHan06/Lop-10A02-Nhom-02#9] [General]

**Comment by NguyenNgocQuynhAnh** [comment]
helu

**Comment by NguyenNgocQuynhAnh** [comment]
phân công sao á

**Reply by GiaHan06** [reply]
đợi mấy bà kia vô rùi tui nói luôn á :((

**Comment by chaudambaotram** [comment]
hế nhon mọi người

**Reply by GiaHan06** [reply]
hiiii

**Comment by GiaHan06** [comment]
ulatr mấy bà kia đâu hết rùi ta

**Comment by LTGHan** [comment]
Haiii

**Reply by GiaHan06** [reply]
Baaaaaa

**Comment by LTGHan** [comment]
Nãy h ik sh clb

**Comment by GiaHan06** [comment]
ủa h 4 câu mấy bạn muốn chia sao???

**Reply by LTGHan** [reply]
Ai bík :))))

**Reply by GiaHan06** [reply]
trả lời zì nghe muốn khóc zạy :((

**Reply by LTGHan** [reply]
😢😢😢

**Comment by GiaHan06** [comment]
omaigotcacbanoisaolaiimlangnhuthechutuibuonlam

**Comment by HuynhLeBaoTran** [comment]
Hí lu mn

**Comment by HuynhLeBaoTran** [comment]
Giờ phân công sao nè^^

**Comment by GiaHan06** [comment]
mấy bạn ơi vào giúp mình đi, làm nhanh xíu rùi tụi mình nghỉ ngơi nè

**Comment by HuynhLeBaoTran** [comment]
Mn mn oi mn đâu roii

**Comment by GiaHan06** [comment]
bh chỉ có phân công với thảo luận chút xíu thui

**Comment by HuynhLeBaoTran** [comment]
Các tềnh êu cụa toy @NguyenNgocQuynhAnh @chaudambaotram @LTGHan

**Reply by GiaHan06** [reply]
h có 4 câu thui bà muốn chia shao nè??

**Reply by chaudambaotram** [reply]
tại tui thấy 2 câu bài 6 phức tạp hơn nên tui nghĩ cho 2 bạn làm 1 câu lun và chỉ cần 1 bạn làm 2 câu bài 5 thui.Mà như vậy tui thấy cũng hơi quá đáng,ko bít đc ko?

**Reply by chaudambaotram** [reply]
mà tui làm cũng gần xong 2 câu bài 5 lun rùi

**Reply by GiaHan06** [reply]
ũa :)) tui chưa chia mà bà làm rùi hã

**Reply by chaudambaotram** [reply]
tại sáng này dậy sớm mà chưa bàn j ới nhau.rảnh quá nên tui ngồi làm trước 1 ít vì câu hỏi bài 5 dễ,chỉ có bài  là tui đọc ko hiểu thui

**Comment by HuynhLeBaoTran** [comment]
Í nhma câu hỏi ở đou zị mn:((

**Reply by GiaHan06** [reply]
ũa hưm bữa tui hỏi thấy hết chưa mà hong ai hỏi gì tui nên tui tưởng mấy bà thấy hết ròi nên tui hong gửi :((

**Reply by GiaHan06** [reply]
sin lõi mấy bà nhìuuuu

**Reply by HuynhLeBaoTran** [reply]
chòi tui phải sin lõi mới đún chứ:((

**Comment by HuynhLeBaoTran** [comment]
Tui hoang mang qa

**Comment by GiaHan06** [comment]
tui thấy câu 4 của bài 6 2 bạn làm được á, tại có 2 cái ví dụ mà mỗi cái phải vừa mô tả bằng sơ đồ khối với liệt kê bước á, rùi vế sau nữa nên 2 bạn làm là ok

**Comment by NguyenNgocQuynhAnh** [comment]
ủa gì mới zo mng phân công sao rồi =))

**Comment by GiaHan06** [comment]
Bh mình chia nè mấy bà ơi, bài 5 mỗi bạn 1 câu còn bài 6 1 bạn câu 3 2 bạn câu 4 được hong?? hay chia như Trâm nói khi nãy

**Reply by HuynhLeBaoTran** [reply]
okk chia như bà đi nè

**Reply by NguyenNgocQuynhAnh** [reply]
vậy để tui làm câu 3 bài 6 ha

**Comment by HuynhLeBaoTran** [comment]
Bóc câu hỏi đi mn oiiii

**Reply by chaudambaotram** [reply]
tui làm cz ần xong câu 1 bài 5 rùi ấy.Tui làm câu đó lun đc ko?

**Reply by HuynhLeBaoTran** [reply]
uki nè bà

**Reply by chaudambaotram** [reply]
thanks mọi người

**Comment by GiaHan06** [comment]
ai muốn làm câu nào thì xí trước đi nè

**Comment by NguyenNgocQuynhAnh** [comment]
giờ mình tự tạo column đúng hok mấy bà

**Reply by GiaHan06** [reply]
đúng ùi

**Comment by HuynhLeBaoTran** [comment]
Zay tui làm câu 2 nhé mn

**Comment by GiaHan06** [comment]
ũa zạy tui với @LTGHan làm câu 4 he

**Comment by GiaHan06** [comment]
sin trào các bạng thân iu của tui

**Comment by NguyenNgocQuynhAnh** [comment]
trào =))

**Comment by GiaHan06** [comment]
ũa mấy bạn kia đou mất rùi trừi

**Reply by LTGHan** [reply]
Hallo

**Comment by HuynhLeBaoTran** [comment]
sin ckao mnnnnn

**Comment by HuynhLeBaoTran** [comment]
khi nào mình bắt đầu làm chung nè

**Comment by NguyenNgocQuynhAnh** [comment]
> Mình không biết là các bạn bị sao ấy nhỉ? Cho mình xin lỗi nếu những lời mình nói là không đúng nhé. Tụi mình đã hẹn với nhau là 7h30 tối sẽ lên đây làm việc thảo luận cùng nhau, mình không ép và cũng đã hỏi ý kiến của các bạn và mọi người cũng đồng ý hết, nên mình cũng không biết vì sao mà có bạn không lên đúng giờ nhỉ? Nếu như có việc bận thì cứ nhắn tin cho mình hoặc nhắn thẳng vào group, đằng này mình chẳng thấy ai nói gì cả. Với lại mình thấy nhóm của tụi mình khá là "lạnh lẽo", có thể là do chưa được trực tiếp gặp mặt nói chuyện với nhau nên là chưa thân lắm ấy, nhiều lúc lên đây chào nhau được mấy câu cái mọi người biến đi đâu mất cái là mình phải gọi mấy bạn lên lại. Nhiều lúc như vậy mình lại sợ các bạn thấy phiền ấy chứ, tại mình nhắn nhiều quá, nhưng mà đâu có ai làm theo như mình nói đâu? Có vài cái tin mình nhắn dặn rất nhiều lần, ừ mấy bạn thả tim là đã xem, có thể do các bạn quên hay là không quan tâm đến, mình cũng không biết nữa, nếu có sai thì mình xin lỗi. Với cả mấy cái bài báo cáo này thầy không những sẽ đánh giá cá nhân mà còn có nhóm nữa ấy, nên tụi mình phải phối hợp, trao đổi và hoạt động tích cực lên chứ, teamwork chứ đâu phải work by yourself đâu. Nhiều lúc mình bị tủi thân lắm ấy chứ, mình cứ nghĩ là tụi mình sẽ làm việc thật vui vẻ với nhau để có thể thân thiết với hiểu nhau hơn, nhưng mọi chuyện lại không diễn ra như vậy... Đó là những lời mình muốn nói với các bạn, không biết là các bạn có thấy không nữa, nhưng mà mình mong là các bạn sẽ hiểu cho mình. Cảm ơn các bạn.

bữa zo xem thấy có 2 3 tin nhắn nên hok bíc nhắn gì luôn :<

**Reply by GiaHan06** [reply]
bởi zì lúc đó tui đang ở quê có công việc, mà hong thấy ai nói gì nên tui cũm hong biết làm gì :((

**Reply by NguyenNgocQuynhAnh** [reply]
hoi chiều ráng zo làm cho xong luôn :33

**Reply by HienCaoPhuNhuan** [reply]
@NguyenNgocQuynhAnh sao thầy tìm quài ko thấy cái bài văn tế này nhỉ

**Reply by GiaHan06** [reply]
@HienCaoPhuNhuan zì zãy thầy:)) chiện nó qua lâu gòi thầy cứ cho qua điii🤓

**Reply by HienCaoPhuNhuan** [reply]
> @HienCaoPhuNhuan zì zãy thầy:)) chiện nó qua lâu gòi thầy cứ cho qua điii🤓

muốn đọc qué, cho mấy đứa làm cái này là để hít hà mà

**Reply by GiaHan06** [reply]
> muốn đọc qué, cho mấy đứa làm cái này là để hít hà mà

ák qua :'(

**Reply by HienCaoPhuNhuan** [reply]
> > muốn đọc qué, cho mấy đứa làm cái này là để hít hà mà
> 
> ák qua :'(

người ta gọi là quan tâm mè

**Comment by LTGHan** [comment]
Yo

**Reply by GiaHan06** [reply]
zì đang học mà :))

**Reply by LTGHan** [reply]
Tehee :))

**Reply by GiaHan06** [reply]
coi chừng lát bị gọi nè :>

**Reply by LTGHan** [reply]
Huh ckưa pị gọi nek :>

**Reply by GiaHan06** [reply]
Do hên thoi😏

**Comment by LTGHan** [comment]
It's time

**Reply by GiaHan06** [reply]
Tại cái khác:)) bên đây nhìu quá sợ rối

**Reply by LTGHan** [reply]
Ok
